.github/workflows/test.yml

```diff
@@ -19,6 +19,39 @@ jobs:
       - name: Run test
         run: test/run
 
+  job-runner:
```

Comment:
Is this meant to be temporary, or do you want to keep this? Run-local's `test_pr()` covers the same job-runner and bots PR test, except it also covers the machinery around it.

(No strong opinion, just curious)

Reply:
I kinda want to keep this... after all, it's "free".  Also no strong opinion, though.

Reply:
Yeah, doesn't hurt.